qiskit/transpiler/__init__.py

```diff
@@ -19,8 +19,403 @@
 
 .. currentmodule:: qiskit.transpiler
 
+Overview
+========
+Transpilation is the process of rewriting a given input circuit to match
+the topoplogy of a specific quantum device, and/or to optimize the circuit
+for execution on present day noisy quantum systems.
+
+Most circuits must undergo a series of transformations that make them compatible with
+a given target device, and optimize them to reduce the effects of noise on the
+resulting outcomes.  Rewriting quantum circuits to match hardware constraints and
+optimizing for performance can be far from trivial.  The flow of logic in the rewriting
+tool chain need not be linear, and can often have iterative sub-loops, conditional
+branches, and other complex behaviors.  That being said, the basic building blocks
+follow the structure given below:
+
+.. image:: /source_images/transpiling_core_steps.png
```

Comment:
Can we change this to something more like:
```
Virtual circuit optimization
Decompose gates acting on 3 or more qubits
Placement on physical qubits ("layout")
Routing on restricted topology
Translate to basis
Physical circuit optimization
```

Reply:
ok